frontend/src/scenes/events/eventsTableLogic.test.ts

```diff
@@ -681,10 +685,27 @@ describe('eventsTableLogic', () => {
             })
 
             it('fires two actions to change state, but just one API.get', async () => {
```

Comment:
This test previously wasn't waiting for the `fetchEventsSuccess` action, so we were checking how many api calls were made before api calls were made. The one that was be counted was from the mount function.

The changes below clean this up.

Reply:
🤦 . Thanks!